examples/browser/wdio.conf.js

```diff
@@ -155,6 +155,7 @@ exports.config = {
     //
     // Gets executed once before all workers get launched.
     onPrepare: function (config, capabilities) {
+        process.argv.push('--root-dir=./test/browsertest');
```

Comment:
I am afraid it is not cross-platform friendly.
I would do something like:
```
process.argv.push(`--root-dir=${path.join(__dirname, 'test', 'browsertest')}`);
```

But the best would be to work in the temp directly.

Reply:
>But the best would be to work in the temp directly.

Do you mean we should use temp.js or something similar to create a temp folder and not have to commit test files to the repo? i.e in the future if we want more custom ui tests with particular filetypes, should we create them only for the duration of the test?

Reply:
I see, so we are generating the actual test files. I do not know then. I would say sure, let's generate them to test, but in the end, one has to debug it, and I do not want to debug things from the temp folder for sure.

Reply:
Ok, for now let's just fix it to make it cross-platform friendly and we can maybe change the test files actual location later.